### DISCUSSION TITLE
Is it possible to access the scada-lts API without using the graphical interface?

### DISCUSSION THREAD

**Discussion by GabrielTec251** [SCADA-LTS/Scada-LTS#2814] [Q&A]
I had a question from a client's information security about the vulnerabilities of the software, the xss ones I managed to get around but he asked me this question, if someone outside the scada without access would be able to invade or make any changes.

**Comment by Limraj** [comment]
Hi @GabrielTec251,
we put a lot of effort into making the application safe, but as a rule our application is used in closed networks, so the level of security in older versions was much lower, but currently we put a lot of effort into this security, which does not exclude some problems. Among other things, we have implemented Spring-security, we are working on improving the application in terms of XSS attacks, the future version 2.8.0 should be more tight in this respect.

Regards.
